### PR TITLE
ci: adds initial terraform files for test resources

### PR DESCRIPTION
**Pull request by mathetake**

### CODE REVIEW

**Comment by mathetake** [comment]
somebody rubber stamping plz

File: .github/terraform/.gitignore
Comment (zirain) [inline .github/terraform/.gitignore:1]: this's fine to move to root folder.

File: .github/terraform/.terraform.lock.hcl
Comment (zirain) [inline .github/terraform/.terraform.lock.hcl:1]: do we really need this file?

File: .github/terraform/.gitignore
Comment (mathetake) [reply .github/terraform/.gitignore:1]: yeah https://developer.hashicorp.com/terraform/language/files/dependency-lock

File: .github/terraform/.terraform.lock.hcl
Comment (mathetake) [reply .github/terraform/.terraform.lock.hcl:1]: yeah https://developer.hashicorp.com/terraform/language/files/dependency-lock

File: .github/terraform/.gitignore
Comment (mathetake) [reply .github/terraform/.gitignore:1]: done

**Review by zirain — APPROVED** [review]

**Pull request merged**